scripts/assets/options.go.templ

```diff
@@ -0,0 +1,197 @@
+<%
+// EJS template
+// Run `npm run generate:cli-options` to update src/go/rdctl/pkg/options/generated/options.go
+
+
+function collectServerSettingsForJSON(settings, indent, lines) {
+  Object.keys(settings).forEach((propertyName) => {
+    collectServerSettingsForJSONProperty(propertyName, settings[propertyName], indent, lines);
+  });
+}
+
+function collectServerSettingsForJSONProperty(propertyName, settings, indent, lines) {
+  const decln = `${ indent }${ propertyName } `;
+
+  if (typeof (settings.type) === 'object') {
+    lines.push(`${ decln } struct {`);
+    collectServerSettingsForJSON(settings.type, `${ indent }  `, lines);
+    lines.push(`${ indent }} \`json:"${ uncapitalize(propertyName) }"\``);
+  } else {
+    lines.push(`${ decln } *${ settings.type } \`json:"${ uncapitalize(propertyName) },omitempty"\``);
+  }
+}
+
+function collectServerSettingsWithoutJSON(settings, indent, lines) {
+  Object.keys(settings).forEach((propertyName) => {
+    collectServerSettingsPropertyWithoutJSON(propertyName, settings[propertyName], indent, lines);
+  });
+}
+
+function collectServerSettingsPropertyWithoutJSON(propertyName, settings, indent, lines) {
+  const decln = `${ indent }${ propertyName } `;
+
+  if (typeof (settings.type) === 'object') {
+    lines.push(`${ decln } struct {`);
+    collectServerSettingsWithoutJSON(settings.type, `${ indent }  `, lines);
+    lines.push(`${ indent }}`);
+  } else {
+    lines.push(`${ decln } ${ settings.type }`);
+  }
+}
+
+function getUsageForCommand(command) {
+  const usageParts = [command.usageNote ?? ''];
+
+  if (command.enums) {
+    usageParts.push(`(Allowed values: [${ command.enums.join(', ')}].)`);
+  }
+  if (command.aliasFor) {
+    usageParts.push(`(Alias for --${ command.aliasFor }.)`);
+  }
+  return usageParts.join(' ').trim();
+}
+
+function getCommandLineArgValue(lcTypeName, capitalizedName) {
+  switch (lcTypeName) {
+  case 'bool':
+    return `+"="+strconv.FormatBool(SpecifiedSettings.${ capitalizedName })`;
+  case 'int':
+    return `, strconv.Itoa(SpecifiedSettings.${ capitalizedName })`;
+    break;
+  default:
+    return `, SpecifiedSettings.${ capitalizedName }`;
```

Comment:
No, because currently the only string settings that can be changed from the command-line have a fixed set of values, and open values like the `imageAllowList` and `WSLIntegrations` currently can't be set from the command-line.